exporter/datasetexporter/logs_exporter_test.go

```diff
@@ -407,3 +403,155 @@ func TestConsumeLogsShouldSucceed(t *testing.T) {
 		addRequest,
 	)
 }
+
+func TestOtelSeverityToDataSetSeverityWithSeverityNumberNoSeverityText(t *testing.T) {
```

Comment:
Consider parametrized test approach to improve readability, eg. https://elliotchance.medium.com/go-data-driven-or-parameterized-tests-cd2b62b21c8a

Reply:
Yeah, I was a thinking about it, but I was a bit torn.

In tests, a lot of times, I personally try to avoid loops and parameterized approach since I've seen many issues over the years where there was a bug in the loop / parameters and the tests didn't actually end up executing all the intended paths / parameters. And code coverage wouldn't really bubble this up either (unless explicitly looking for it).

So while non-parameterized / non-loop approach does indeed mean more duplication, it's usually easier to spot issue like that.

Reply:
I am not that experienced with GO, but my goal is to improve readability of test, see eg java and junit5 aproach `@ValueSource and @ParametrizedTests` eg https://www.baeldung.com/parameterized-tests-junit-5#first-impression this provides very nice readability without any loops.

We might at leas split the test so its bit readable eg. ```TestMappingOtelSeverityNumberAndNoTextToDataSetDebugSeverit, TestMappingOtelSeverityNumberAndNoTextToDataSetTraceSeverity and TestMappingInvalidOtelSeverityNumberAndNoTextToDataSetDefaultSeverity```

Reply:
Done.